justfile

```
@@ -228,7 +228,8 @@ _compile-android target:
     cd "$RUST_WORKSPACE"; \
         cargo ndk --bindgen -t $(echo "{{target}}" | sed 's/[^ ]* */&/g') -p $ANDROID_PLATFORM_VERSION \
         -o "{{justfile_directory()}}/$FLUTTER_WORKSPACE/android/src/main/jniLibs" build \
-        --release -p xayn-discovery-engine-bindings --locked
+        --release \
+        -p xayn-discovery-engine-bindings --locked
```

:art: if we split the line by arguments then we should do so completely
```suggestion
        --release \
        -p xayn-discovery-engine-bindings \
        --locked
```
this would also apply for l.229

---

done, also rebased to resolve merge conflicts.